modules/indexer/issues/indexer.go

```diff
@@ -143,6 +143,14 @@ func InitIssueIndexer(syncReindex bool) {
 		var populate bool
 		switch setting.Indexer.IssueType {
 		case "bleve":
+			defer func() {
+				if err := recover(); err != nil {
+					log.Error("PANIC whilst initializing issue indexer: %v\nStacktrace: %s", err, log.Stack(2))
+					log.Error("The indexer files are likely corrupted and may need to be deleted")
+					holder.cancel()
+					log.Fatal("PID: %d Unable to initialize the Bleve Issue Indexer at path: %s Error: %v", os.Getpid(), setting.Indexer.IssuePath, err)
```

Comment:
Since we don't have a `delete indexes` command (yet?), could we also print something like "You can completely remove the %s directory to make Gitea recreate the indexes".

Reply:
done